discussion/app/views/fragments/commentShare.scala.html

```diff
@@ -1,7 +1,10 @@
-@(comment: Comment)
+@import _root_.model.{Facebook, ShareLinks, SharePlatform, Twitter}
```

Comment:
In this case it is - the discussion project [has a number of default imports](https://github.com/guardian/frontend/blob/master/project/Frontend.scala#L104) that would otherwise cause `model._` to clash. I haven't changed this as each project can do what they want